tools/openvino_dev/setup.py

```diff
@@ -206,16 +206,33 @@ def get_description(desc_file_path):
     ]
 
 
+def concat_files(output_file, input_files):
+    with open(output_file, 'w', encoding='utf-8') as outfile:
+        for filename in input_files:
+            with open(filename, 'r', encoding='utf-8') as infile:
+                content = infile.read()
+                outfile.write(content)
+    return output_file
+
+
+description_md = SCRIPT_DIR.parents[1] / 'docs' / 'install_guides' / 'pypi-openvino-dev.md'
+md_files = [description_md, SCRIPT_DIR.parents[1] / 'docs' / 'install_guides' / 'pre-release-note.md']
+
+if(os.getenv('CI_BUILD_DEV_TAG')):
+    output = Path.cwd() / 'build' / 'pypi-openvino-dev.md'
+    output.parent.mkdir(exist_ok=True)
+    description_md = concat_files(output, md_files)
+
 setup(
     name='openvino-dev',
     version=os.getenv('OPENVINO_VERSION', '0.0.0'),
-    author='Intel® Corporation',
-    license='OSI Approved :: Apache Software License',
-    author_email='openvino_pushbot@intel.com',
-    url='https://docs.openvino.ai/latest/index.html',
-    download_url='https://github.com/openvinotoolkit/openvino/tags',
-    description='OpenVINO(TM) Development Tools',
-    long_description=get_description(SCRIPT_DIR.parents[1] / 'docs/install_guides/pypi-openvino-dev.md'),
+    author=os.getenv('WHEEL_AUTHOR', 'Intel® Corporation'),
+    license=os.getenv('WHEEL_LICENCE_TYPE', 'OSI Approved :: Apache Software License'),
+    author_email=os.getenv('WHEEL_AUTHOR_EMAIL', 'openvino_pushbot@intel.com'),
+    url=os.getenv('WHEEL_URL', 'https://docs.openvino.ai/latest/index.html'),
```

Comment:
should not we refer to prerelease version of documentation? or at least to nightly

Reply:
Yes, it makes sense to use nightly docs revision for pre-release packages.